lua/lazy/core/plugin.lua

```diff
@@ -375,7 +375,9 @@ function M.load()
   -- load specs
   Util.track("spec")
   Config.spec = Spec.new()
-  Config.spec:parse({ vim.deepcopy(Config.options.spec), { url = "https://github.com/folke/lazy.nvim.git" } })
+  Config.spec:parse(
```

Comment:
Since we're changing it anyway, just make it a regular spec, so:

```lua
  Config.spec:parse({ vim.deepcopy(Config.options.spec), { "folke/lazy.nvim" } })
```

Should still work